rllib/core/rl_module/encoder.py

```diff
@@ -63,7 +74,7 @@ def __init__(self, config: EncoderConfig) -> None:
         self._output_spec = self.output_spec()
 
     def get_inital_state(self):
-        raise []
+        return []
 
     def input_spec(self):
         return ModelSpec()
```

Comment:
ModelSpec() is suppose to get updated to SpecDict if we merge my PR first. :) make sure to update if you merge master after that merge happens.